Add plot of pendulum transition

Rojetto opened the pull request.

Rojetto committed:
Add plot of pendulum transition

cknoll reviewed (commented):
Das gefällt mir schon sehr gut. :+1: 

Ich würde noch folgende Änderungen anregen:

- [x] Zwiebelgrafik über die Zeitverläufe,
- [x] \psi -> \varphi (Konsistenz mit \[1])
- [x] Höhe der Zeitverläufe halbieren
- [x] Achsbeschriftung auf deutsch
- [x] uoo -> \texttt{"UOO"} analog der bisherigen Bezeichnung 

\[1]: https://nbviewer.jupyter.org/github/cknoll/demo-material/blob/main/underactuated_systems/triple_pendulum_with_modeltools_plus_simulation-en.ipynb (Das soll unter https://tud.link/7rsv verlinkt werden)

Rojetto committed:
Improve swingup plot

Position onion skin graphic more prominently
Relabel angles to varphi
Relabel (and rescale) x and y axes

Rojetto commented:
Ist eingearbeitet!

Rojetto committed:
Rechteck für Wagenposition
